src/forms/progressive-profiling-popup/index.jsx

```diff
@@ -183,6 +190,7 @@ const ProgressiveProfilingForm = () => {
       <h1
         className="display-1 font-italic text-center mb-4"
         data-testid="progressive-profiling-heading"
+        ref={countryFeildErrorRef}
```

Comment:
This element is not country field so can we rename the ref name or can we set ref on the country field?

Reply:
I renamed it